stores/dotenv/store.go

```diff
@@ -141,7 +141,12 @@ func (store *Store) EmitPlainFile(in sops.TreeBranches) ([]byte, error) {
 		if comment, ok := item.Key.(sops.Comment); ok {
 			line = fmt.Sprintf("#%s\n", comment.Value)
 		} else {
-			value := strings.Replace(item.Value.(string), "\n", "\\n", -1)
+			value, ok := item.Value.(string)
+			if !ok {
+				value = fmt.Sprintf("%v", item.Value)
+			}
+
+			value = strings.ReplaceAll(value, "\n", "\\n")
```

Comment:
The replace only makes sense for strings:
```suggestion
			} else {
				value = strings.ReplaceAll(value, "\n", "\\n")
			}
```